### PR TITLE
Fix gitjob log messages for controller and webhook

### PR DESCRIPTION
**Pull request by manno**
Refers to #2345

### CODE REVIEW

File: internal/cmd/controller/reconciler/cluster_controller.go [hunk 1]
```diff
@@ -170,8 +169,12 @@ func (r *ClusterReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
 	}
 
 	if allReady && cluster.Status.ResourceCounts.Ready != cluster.Status.ResourceCounts.DesiredReady {
-		logrus.Debugf("Cluster %s/%s is not ready because not all gitrepos are ready: %d/%d, enqueue cluster again",
-			cluster.Namespace, cluster.Name, cluster.Status.ResourceCounts.Ready, cluster.Status.ResourceCounts.DesiredReady)
+		logger.V(1).Info("Cluster is not ready, because not all gitrepos are ready",
+			"namespace", cluster.Namespace,
+			"name", cluster.Name,
+			"ready", cluster.Status.ResourceCounts.Ready,
+			"DesiredReady", cluster.Status.ResourceCounts.DesiredReady,
```
Comment (weyfonk) [inline internal/cmd/controller/reconciler/cluster_controller.go:176]: Just curious: is there a rationale for upper- vs lowercase first chars of log fields here?
Are they deliberately inconsistent between themselves for the sake of keeping `DesiredReady` consistent with the state name it refers to?

File: pkg/git/lsremote.go [hunk 2]
```diff
@@ -158,14 +163,14 @@ func (g *git) remoteSHAChanged(branch, sha string) (bool, error) {
 
 	client, err := g.httpClientWithCreds()
 	if err != nil {
-		logrus.Warnf("Problem creating http client to check git remote sha of repo [%v]: %v", g.URL, err)
+		g.log.Error(err, "Problem creating http client to check git remote sha of repo", "url")
```
Comment (weyfonk) [inline pkg/git/lsremote.go:166]: Do we need an additional `"url"` parameter there, given that it's already provided in `g.log`? Plus, the signature of `Error` suggests that we should provide key/value _pairs_. Or am I missing something?
```suggestion
		g.log.Error(err, "Problem creating http client to check git remote sha of repo")
```

File: internal/cmd/controller/reconciler/cluster_controller.go (hunk 1 above)
Comment (manno) [reply internal/cmd/controller/reconciler/cluster_controller.go:176]: Right, should be `desiredReady`.

Comment (manno) [reply internal/cmd/controller/reconciler/cluster_controller.go:176]:
```suggestion
			"desiredReady", cluster.Status.ResourceCounts.DesiredReady,
```

**Review by weyfonk — APPROVED** [review]

**Pull request merged**